<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Balance</class>
 <widget class="QDialog" name="Balance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #A3BAA9;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">object-fit: cover;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: transparent;
border: 1.5px solid #1D1B20;
border-radius: 8;
font-size: 16px;
padding-right:50px;
 font-weight: bold;
}

QPushButton:hover {
	border: none;
}
</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fileImg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>364</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: transparent;
border:none;
border-radius: 8;
font-size: 16px;
padding-left:50px;
 font-weight: bold;
}

QPushButton:hover {
	border: 1.5px solid #1D1B20;
}
</string>
   </property>
   <property name="text">
    <string>Transaction History</string>
   </property>
  </widget>
  <widget class="QLabel" name="dollarSign">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>434</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="payImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>428</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: transparent;
border:none;
border-radius: 8;
font-size: 16px;
padding-left:50px;
 font-weight: bold;
}

QPushButton:hover {
	border: 1.5px solid #1D1B20;
}
</string>
   </property>
   <property name="text">
    <string>Payment Transfer</string>
   </property>
  </widget>
  <widget class="QLabel" name="Card">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>497</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>492</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: transparent;
border:none;
border-radius: 8;
font-size: 16px;
padding-left:50px;
 font-weight: bold;
}

QPushButton:hover {
	border: 1.5px solid #1D1B20;
}
</string>
   </property>
   <property name="text">
    <string>Card</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>1000</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Example Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>1020</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>example@gmail.com</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>987</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="account">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>1002</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logOut">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>1002</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>690</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Example Name</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Home</zorder>
  <zorder>Logo</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>fileImg</zorder>
  <zorder>payImg</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>dollarSign</zorder>
  <zorder>Card</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>line</zorder>
  <zorder>account</zorder>
  <zorder>logOut</zorder>
  <zorder>label_username</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
